README.md

```diff
@@ -79,16 +78,16 @@ Application's main view allows the user to set which sensors to use and the corr
 
 ### Notifications
 
-When one of the sensors is triggered (reaches the sensibility threshold) a notifications is sent through the following channels (if enabled).
+When one of the sensors is triggered (reaches the sensibility threshold) a notifications are sent through the following channels (if enabled).
```

Comment:
Should be "threshold) a notification is sent" or "threshold) notifications are sent".  I would just drop the "a". I think it makes more sense.